testing, stories, security

aretecode opened the pull request.

aretecode committed:
i18n/config: turn off debug

aretecode committed:
tests/config: add dom & es2019 to tsconfig

aretecode committed:
components: Media - move ScribeIcon out of welcome page

aretecode committed:
deps: add "babel-plugin-require-context-hook”

used for testing

https://github.com/storybookjs/storybook/issues/3406

aretecode committed:
components: Form - split from index, add Story

aretecode committed:
components: Flags - add wcag titles

aretecode committed:
tests: setup storyshots

aretecode committed:
styling: remove letter spacing on links

aretecode committed:
! stories: footer, device, avatar, link, logo, fam, advice

aretecode committed:
! tests: LanguageSwitcher

aretecode committed:
pages: fix typo import

aretecode committed:
! server: change secret to env & fix import of user

and some config files in prep for own repo

aretecode committed:
[large diff] tests: snapshots

aretecode committed:
components: PageHeader

for family & advice reuse
has story

aretecode committed:
styles: cleanup white color again, move styled logo

aretecode committed:
cleanup: unused line in language switcher as todo

aretecode committed:
docs: update description, update todos

aretecode committed:
features: Family - add email to add family form

aretecode committed:
[large diff] snapshots

mfrohberg reviewed (approved):
What broke that required the need to reimplement the require context?

aretecode committed:
!! server: setup so it can run on local & deploy

aretecode committed:
!!! setup auth0 client + profile page

aretecode committed:
docs: update running dev server & todos

aretecode committed:
safety: add catch on fetch

aretecode committed:
safety: default form value

to avoid controlled uncontrolled switch

aretecode committed:
! pages: Credits page

and update todos

aretecode committed:
i18n open source/credits page title

aretecode committed:
styling: responsive footer

now that we have more links

aretecode committed:
[big diff] snapshots

understood-devops-2 committed:
helper + set trust proxy at top

understood-devops-2 committed:
yarnage

aretecode committed:
images: use stock photos, minor footer styles

padding & border on footer
use stock images from sirv & use webp format

aretecode committed:
Merge branch 'data-cleanup' of github.com:aretecode/konsilos-v2-help into data-cleanup

aretecode committed:
formatting: server format (autofix)

aretecode committed:
addFamily connected to state, added uuid

understood-devops-2 committed:
add backend lib

understood-devops-2 committed:
Merge branch 'data-cleanup' of github.com:aretecode/konsilos-v2-help into data-cleanup

aretecode committed:
! config: add back storybook static deploy at /ui

and a minor padding style on device story

aretecode committed:
Merge branch 'data-cleanup' of github.com:aretecode/konsilos-v2-help into data-cleanup

aretecode committed:
move empty avatar story to second

aretecode committed:
config: add back strick around AppStyles in story

aretecode committed:
docs: todos - add some test situations

aretecode committed:
config: storybook missed imports

aretecode committed:
[large diff] snapshots

aretecode committed:
cleanup: remove unused imports

understood-devops-2 committed:
add correct lib

understood-devops-2 committed:
Merge branch 'data-cleanup' of github.com:aretecode/konsilos-v2-help into data-cleanup

aretecode committed:
components: Image (for lazy loading)

minor perf

understood-devops-2 committed:
wip

aretecode committed:
minor seo, perf, wcag, github credit link, storybook link

aretecode committed:
Merge branch 'data-cleanup' of github.com:aretecode/konsilos-v2-help into data-cleanup

aretecode committed:
seo - titles for each page

understood-devops-2 committed:
better

understood-devops-2 committed:
debuggable...

aretecode committed:
! config: move env to env.js file & @ security tags

also
- graphql playground in production
- update workbox config

aretecode committed:
grpahql: add “everything” query

aretecode committed:
Merge branch 'data-cleanup' of github.com:aretecode/konsilos-v2-help into data-cleanup

# Conflicts:
#	src/graphql/resolvers.ts

aretecode committed:
docs: + features section, + graphql playground link

aretecode committed:
typings: process.env typings from our env config

understood-devops-2 committed:
closer

understood-devops-2 committed:
closey

aretecode committed:
graphql: create helper to simplify resolvers

aretecode committed:
graphql: tweak query prop order

aretecode committed:
graphql: fix typo on apiKey env

aretecode committed:
Merge branch 'data-cleanup' of github.com:aretecode/konsilos-v2-help into data-cleanup

# Conflicts:
#	src/graphql/resolvers.ts

aretecode committed:
docs: todo - add pagination todo

aretecode committed:
graphql: add transform helper fn

understood-devops-2 committed:
adding foreignKey for user on FamilyMember

understood-devops-2 committed:
Merge branch 'data-cleanup' of github.com:aretecode/konsilos-v2-help into data-cleanup

aretecode committed:
graphql: typings to list results

aretecode committed:
!!! graphql: apollo hooks to get & add everything

- create graphql queries & mutations for api
- create (_strongly typed_) use`X` hooks for our endpoints
- add todos for remaining endpoints
- these need to be tested, and they should be easy to test since the queries are easily separated and such
- ! console.log the data given back from the hooks until they are *hooked* up :P
- create GraphqlAddFamily & GraphqlFamilyList (same for advice) to wrap the existing AddFamily & FamilyList (which are “dumb” components). this is similar to how you “wire” up a component in redux or connect it observably in mobx.

aretecode committed:
fix extra commas in footer

understood-devops-2 committed:
laid out

understood-devops-2 committed:
wip

aretecode committed:
!!! i18n - use nextjs version

aretecode committed:
yarnage

aretecode committed:
config: safety in env file

aretecode committed:
!! auth0 - fix redirect so /callback => /user => /

aretecode committed:
Merge pull request #15 from aretecode/i18n-next

I18n next

aretecode committed:
Merge branch 'data-cleanup' of github.com:aretecode/konsilos-v2-help into i18n-next

# Conflicts:
#	src/graphql/resolvers.ts

aretecode committed:
graphql - update some of the names, add examples

aretecode committed:
Merge branch 'merge' of github.com:aretecode/konsilos-v2-help into data-cleanup

aretecode committed:
Merge branch 'graphql-hooks' of github.com:aretecode/konsilos-v2-help into data-cleanup

aretecode commented:
this ended up being quite the hefty PR, merge branch v2 😄

aretecode committed:
! config: remove storybook from now again

added back to todo

aretecode committed:
config: nextjs - fix config & deps for weird bug

https://github.com/zeit/now-builders/issues/61

aretecode committed:
[large diff] snapshots

aretecode committed:
typings: fix type callings in utils/is

aretecode committed:
…use old i18n

- narrowing down scope for now__import error
- use old i18n
- remove util imports
